### DISCUSSION TITLE
Scrolling in Note Panel is not smooth

### DISCUSSION THREAD

**Discussion by rubisara38** [freeplane/freeplane#764] [Help]
When I store long content in the note of a node, and then I scroll inside the Note panel to read that content, it feels jerky and not smooth. 

This can be clearly seen when you compare the scrolling experience of Note Panel and that of Chrome, Firefox or even Notepad++

How can I make the scrolling in Note Panel more smooth?

Thank you!

**Comment by euu2021** [comment]
Try playing with that setting: `Preferences…->Behaviour->Scrollbar->Speed`

**Reply by rubisara38** [reply]
@euu2021 I've tried that setting before but it did nothing to the note panel's scroll smoothness. Perhaps it is for another component of Freeplane.
